packages/neos-ui-redux-store/src/UI/ContentCanvas/index.js

```diff
@@ -13,6 +13,7 @@ const SET_PREVIEW_URL = '@neos/neos-ui/UI/ContentCanvas/SET_PREVIEW_URL';
 const SET_SRC = '@neos/neos-ui/UI/ContentCanvas/SET_SRC';
 const FORMATTING_UNDER_CURSOR = '@neos/neos-ui/UI/ContentCanvas/FORMATTING_UNDER_CURSOR';
 const SET_CURRENTLY_EDITED_PROPERTY_NAME = '@neos/neos-ui/UI/ContentCanvas/SET_CURRENTLY_EDITED_PROPERTY_NAME';
+const DOCUMENT_INITIALIZING = '@neos/neos-ui/UI/ContentCanvas/DOCUMENT_INITIALIZING';
```

Comment:
Ok, so you suggest to use Sagas to transcend reducer boundaries? Before we mainly tried to use Sagas for async actions.
So I see these options:
1) Ignore sub-reducer boundaries and allow to modify all state from any reducer. That would make a lot of logic very compact and explicit, also cleanup event log and give more semantical meaning to actions, that they are no longer a mere setters. On the other hand that would make the whole application a monolith.
2) Fire multiple actions straight from the create node dialog. That makes the logic scattered all over the place and action log noisy.
3) Use sagas to connect separate sub-reducer states by re-throwing actions based on other actions. The action log will still be noisy, but at least this logic is expressed in one place somewhere.

I myself would probably prefer 1), but it looks like we are drifting in the direction of 3) and use actions as mere setters, so maybe we should not resist the flow and go this way that feels natural to you.

Wdyt?